xbmc/cores/AudioEngine/Sinks/AESinkAUDIOTRACK.cpp

```diff
@@ -819,33 +822,36 @@ void CAESinkAUDIOTRACK::EnumerateDevicesEx(AEDeviceInfoList &list, bool force)
         if (CJNIAudioFormat::ENCODING_DOLBY_TRUEHD != -1)
           m_info.m_streamTypes.push_back(CAEStreamInfo::STREAM_TYPE_TRUEHD);
       }
-      // Android v24 can do real IEC API
-      if (CJNIAudioManager::GetSDKVersion() >= 24)
+      // Android v24 and backports can do real IEC API
+      if (CJNIAudioFormat::ENCODING_IEC61937 != -1)
```

Comment:
This one is actually unrelated to AML, isn't it?
Amazon or others might decide one day to backport ENCODING_IEC61937...